manager/testdata/e2e/module-implicit-copy-kafka-to-s3-stream.yaml

```diff
@@ -82,5 +82,5 @@ spec:
     - id: "encrypt-ID"
       name: "encrypt"
       level: 2 # column
-  chart: ghcr.io/the-mesh-for-data/m4d-kafka:0.1.0
+  chart: kind-registry:5000/m4d-system/m4d-kafka:0.1.0
```

Comment:
A read module chart is not pushed to kind-registry. Will it be a problem to access ghcr.io for this chart?

Reply:
I think for the read module it's fine to pull it from ghcr.io as it's coming from a different repo anyways. For the modules that are maintained in this repository it's important to test the local version here so we test possible changes.

Reply:
for development purposes, I really want to ability to run my development helm chart (from local repo), but I also understand the need to have the sample work w/o a local repo ... so the only way this can work is using templating, if we wrap our samples (all together) as a helm package then (replacing helm repo) would be easy.